mitodl/mitxpro
# chore: switch migrations to the release phase

### What are the relevant tickets?
[#4230](https://github.com/mitodl/hq/issues/4230)

### Description (What does it do?)
This PR switches `bin/post_compile migration` script to the release phase script.

### Screenshots (if appropriate):


### How can this be tested?
It will be tested in heroku deployment on RC after merging.
For local testing: 
  -  Go to the web bash shell
  - Run `bash scripts/heroku-release-phase.sh`
  - Verify that the script containing migration commands runs successfully.

### Additional Context
